src/web_app/js/peerconnectionclient.js

```diff
@@ -319,6 +322,18 @@ PeerConnectionClient.prototype.onIceConnectionStateChanged_ = function() {
   if (this.pc_.iceConnectionState === 'completed') {
     trace('ICE complete time: ' +
         (window.performance.now() - this.startTime_).toFixed(0) + 'ms.');
+    // Associate SSRC's with media stream tracks and user ID in the
```

Comment:
Did not event think of the other ICE states ;). We need both localDescription and remoteDescription SDP with SSRC's before we can make the call and this seems the best place (if not the only place) in peerconnectionclient.js